plugins/poll/assets/javascripts/discourse/components/poll-results-tabs.gjs

```diff
@@ -0,0 +1,74 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { fn } from "@ember/helper";
+import { on } from "@ember/modifier";
+import { action } from "@ember/object";
+import { eq } from "truth-helpers";
+import I18n from "discourse-i18n";
+import PollResultsIrv from "./poll-results-irv";
+import PollResultsStandard from "./poll-results-standard";
+
+const tabOne = I18n.t("poll.results.tabs.votes");
+const tabTwo = I18n.t("poll.results.tabs.outcome");
```

Comment:
can probably be inside class definition:

tabOne = I18n.t("poll.results.tabs.votes");
tabTwo = I18n.t("poll.results.tabs.outcome");

this.tabOne
this.tabTwo

Reply:
Done.